src/views/modelePageAccueil.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('node:fs');
+
+class ModelePageAccueil {
+  infosUtilisateur;
+
+  constructor(infosUtilisateur) {
+    this.infosUtilisateur = infosUtilisateur;
+    this.modeleVierge = fs.readFileSync(`${__dirname}/pageAccueil.html`).toString();
+  }
+
+  enHTML = () => (this.infosUtilisateur
+    ? this.avecUtilisateurConnecte()
+    : this.sansUtilisateurConnecte());
+
+  avecUtilisateurConnecte() {
+    return this.modeleVierge.replace(
+      '{{contenu}}',
+      `<p>Utilisateur courant : ${this.infosUtilisateur.given_name} ${this.infosUtilisateur.family_name}</p>
```

Comment:
On ne garantit pas que `this.infosUtilisateur` contient un attribut `given_name` ou `famiily_name`. On risque ici de se retrouver avec une erreur, dont on pourrait se prémunir (par exemple, en écrivant `this.infosUtilisateur?.given_name` ou en ajoutant un `catch` dans la route).

Reply:
Vu ensemble – en fait, cette méthode n'est appelée que s'il y a des `infosUtilisateur` (cf. ligne 11). Éventuellement, mettre une sentinelle, mais c'est peut-être overkill.